main.go

```diff
@@ -98,19 +84,20 @@ func init() {
 	-g dockerfile: only the Dockerfile
 	-g docker-compose: only the docker-compose.yml + Dockerfile
 	-g service: only the service.yml + Dockerfile (cloud 66 specific)
-	-g dockerfile,service,docker-compose (all files)
+	-g skycap: only the skycap files (cloud66 specific)
```

Comment:
Cloud 66

Reply:
There is no need to have this singled out, `skycap` is an option for `g` like the other ones.